sentry-core/src/main/java/io/sentry/core/HubWrapper.java

```diff
@@ -0,0 +1,22 @@
+package io.sentry.core;
+
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
+
+public abstract class HubWrapper implements IHub {
```

Comment:
maybe a better name for this class?!

Reply:
The same concept exists in [C# and there's it's called `HubAdapter`](https://github.com/getsentry/sentry-dotnet/blob/385a1e4b9f550c1382f14ddb720a1fa85a26e80e/src/Sentry/Extensibility/HubAdapter.cs). I personally prefer Adapter over Wrapper but they are two names for the same pattern.